pep-0701.rst

```diff
@@ -320,7 +342,76 @@ All restrictions mentioned in the PEP are lifted from f-string literals, as expl
+  on simple mechanisms to detect strings and f-strings, such as regular expressions or simple delimiter
+  matching tools. Introducing quote reuse in f-strings will either make it trickier to keep these tools
+  working or will break the tools altogether (as, for instance, regular expressions cannot parse arbitrary nested
+  structures with delimiters). The IDLE editor, included in the standard library, is an example of a
+  tool which may need some work to correctly apply syntax highlighting to f-strings.
+
+Here are some of the arguments in favour:
+
+* Many languages that allow similar syntactic constructs (normally called "string interpolation") allow quote
+  reuse and arbitrary nesting. These languages include JavaScript, Ruby, C#, Bash, Swift and many others.
+  The fact that many languages allow quote reuse can be a compelling argument in favour of allowing it in Python. This
+  is because it will make the language more familiar to users coming from other languages. 
+
+* As many other popular languages allow quote reuse in string interpolation constructs, this means that editors
+  that support syntax highlighting for these languages will already have the necessary tools to support syntax
+  highlighting for f-strings with quote reuse in Python. This means that although the files that handle syntax
+  highlighting for Python will need to be updated to support this new feature, is not expected to be impossible
+  or very hard to do.
+
+* One advantage of allowing quote reuse is that it composes cleanly with other syntax. Sometimes this is referred to
+  as "referential transparency". An example of this is that if we have ``f(x+1)``, assuming ``a`` is a brand new variable, it
+  should behave the same as ``a = x+1; f(a)``. And vice versa.  So if we have::
+
+    def py2c(source):
+        prefix = source.removesuffix(".py")
+        return f"{prefix}.c"
+
+  It should be expected that if we replace the variable ``prefix`` with its definition, the answer should be the same::
+
+    def py2c(source):
+        return f"{source.removesuffix(".py")}.c"
+
+* Limiting quote reuse will considerably increase the complexity of the implementation of the proposed changes. This is because
+  it will force the parser to have the context that is parsing an expression part of an f-string with a given quote in order
+  to know if it needs to reject an expression that reuses the quote. Carrying this context around is not trivial in parsers that
+  can backtrack arbitrarily (such as the PEG parser). The issue becomes even more complex if we consider that f-strings can be
+  arbitrarily nested and therefore several quote types may need to be rejected.
+
+To gather feedback from the community,
+ `a poll <https://discuss.python.org/t/pep-701-syntactic-formalization-of-f-strings/22046/24>`__
+has been initiated to get a sense of how the community feels about this aspect of the PEP.
```

Comment:
```suggestion
  To gather feedback from the community,
  `a poll <https://discuss.python.org/t/pep-701-syntactic-formalization-of-f-strings/22046/24>`__
  has been initiated to get a sense of how the community feels about this aspect of the PEP.
```

Seems like the indent got messed up somehow here as a result of my suggestion and results in the build failing; this should fix it.